### ISSUE TITLE
Unable to Launch Python 3.8

### ISSUE THREAD

**Issue by reillysiemens** [nushell/nushell#2078]
**Describe the bug**
Python 3.8 cannot be launched from `nu`. Attempting to launch it results in
```
❯ python
error: Command not found
  ┌─ shell:1:1
  │
1 │ python
  │ ^^^^^^ command not found
```

It _can_, however, be launched from PowerShell, which leads me to believe there's something amiss here.

**To Reproduce**
1. Install Nu on Windows with `cargo install -f nu --features=stable`.
2. Install [Python 3.8 from the Microsoft Store](https://www.microsoft.com/en-us/p/python-38/9mssztt1n39l?activetab=pivot:overviewtab).
3. Run `nu`.
4. Attempt to launch `python`, `python3`, or `python3.8`.

**Expected behavior**
I expect the Python 3.8 interpreter to be launched.

**Screenshots**

Windows Terminal 1.0.1401.0
<img width="689" alt="python-command-not-found" src="https://user-images.githubusercontent.com/3084059/85969781-7a2c5e00-b97d-11ea-90e9-0bc6c5dd582d.png">

PowerShell
<img width="961" alt="powershell-python" src="https://user-images.githubusercontent.com/3084059/85969783-7dbfe500-b97d-11ea-94eb-09c66eba2aad.png">

**Configuration:**
 - OS: Windows 10 Pro 1903 Build 18362.900
 - Version: 0.15.0
 - Features: Stable

**Comment by jonathandturner** [comment]
I believe this is fixed in the upcoming 0.16.0 release as part of https://github.com/nushell/nushell/pull/2035

The 0.16.0 release should be out tomorrow, or you could try the latest source.

**Comment by reillysiemens** [comment]
Huzzah! It works! Thanks! 🎉

<img width="676" alt="nushell-python" src="https://user-images.githubusercontent.com/3084059/86351652-74eb3f80-bc19-11ea-9bf8-3ce0a5f04956.png">
